src/readers/earnings.ts

```diff
@@ -67,6 +75,42 @@ export class EarningsReader<C extends ChainId> extends Reader<C> {
     return result;
   }
 
+  async accountEarnings(accountAddress: Address): Promise<AccountEarnings[]> {
```

Comment:
Just realised this logic is wrong - needs to take into account deposts/withdrawals/transfers